test/api/buffer/receivers/delete/001-create-ok/expected-response.json

```diff
@@ -0,0 +1,6 @@
+{
+  "receiverId": "delete-receiver",
+  "name": "delete-receiver",
+  "url": "https://buffer.keboola.local/v1/import/delete-receiver/#/%s"
```

Comment:
Drobnost, ak by si namiesto `delete-receiver`, pouzival `foo`, alebo `my-receiver`, tak by bolo hned jasne, ze je to receiver ID.

Reply:
Ty receivery se mažou až doběží všechny testy, takže používám míň generický jména

Reply:
> Ty receivery se mažou až doběží všechny testy
Nebezi kazdy test vo vlastnom namespace. Ci si myslel, ked dobehnu vsetky requesty?

Reply:
> Nebezi kazdy test vo vlastnom namespace

Neběží, ten `Cleanup` je v `RunApiServer`

Reply:
A API server sa spusta novy pre kazdy test (?)

Reply:
Mělo by to generovat namespace pro každou sadu requestů. Nerozumím tomu, co se bere jako jeden "functional test". Měl jsem konflikty mezi např. `receivers/create` a `receivers/delete`.

Reply:
tak bude asi niekde chyba, funkcionalny test je jeden adresar, v ktorom je viacero `requests`, z kodu by to malo byt pekne vidiet. 

Ok, ak nevies preco tam boli konflikty, tak neries a pozriem sa na to.

Reply:
( tj `functional test`  je `sadu requestů`, ktora bezi oproti nejakej instancii API)

Dopln prosim komentare, kde ti to nie je jasne.

Reply:
Jen ta terminologie mě zmátla, `GetTestDirs` mluví o `[category]/[test]` a category nikde jinde zmíněný není

Reply:
Ok, tak to prosim uprav aby to bolo lepsie vysvetlene, ... `category` su tie `receivers` a je to nejaka `group`, aby sme nemali vsetky testy v jednom adresary (vid CLI, tam sme to tak mali, ale uz sa v tom nedalo orientovat).

Reply:
Upravil jsem a přidal jsem tam example.